spec/system/distribution_system_spec.rb

```diff
@@ -699,6 +699,34 @@
     end
   end
 
+  context "with dropdown options changed while open", skip: "select2 not working correctly with cuprite" do
```

Comment:
Really don't understand why I can't get this system spec to work. These steps work in a browser locally. I notice some weird behavior with select2 though. It seems like those events aren't being triggered in cuprite.

Reply:
I should probably just delete this?

Reply:
@coalest if you can't get the spec working, it's probably best to delete it, yes. Otherwise it'll sit around forever. :(

Reply:
Deleted it, as I've already spent a long time trying to get it work. I suspect it's related to that open cuprite issue I linked about, but haven't confirmed.